authors/apps/articles/tests/test_articles.py

```diff
@@ -99,3 +104,69 @@ def test_delete_article(self):
         res = self.client.delete(
             self.url+slug+'/',  format='json')
         self.assertEqual(res.status_code, status.HTTP_200_OK)
+
+    def test_rate_article_you_created(self):
+        """
+        Test to check whether you can rate an article you created
+        """
+        self.client.credentials(
+            HTTP_AUTHORIZATION='Bearer ' + self.get_user_token())
+        response = self.client.post(
+            self.url, data=self.article, format='json')
+        slug = response.data.get('slug')
+        rate_url = reverse('article:ratings_list', args=[slug])
+        self.client.credentials(
+            HTTP_AUTHORIZATION='Bearer ' + self.get_user_token())
+        response = self.client.post(
+            rate_url, data=self.rate, format='json')
+        self.assertEqual(
+            response.data['error'], 'You cannot rate an article that you created.')
```

Comment:
use pep8 standards to make your lines of code shorter

Reply:
Done